tox.ini

```diff
@@ -11,3 +11,9 @@ ignore = E126,E127
 commands = flake8
 deps = flake8
 basepython = python3.4
+
+[testenv:discover]
+commands = nosetests {posargs} {toxinidir}
+deps =
+    -rrequirements.txt
```

Comment:
It looks very wrong at first glance but per https://media.readthedocs.org/pdf/tox/latest/tox.pdf it's correct. Super weird